### PR TITLE
add flux facts to main page

### PR DESCRIPTION
**Pull request by dholbach**
This is a quick draft of some of the Flux facts, @scottrigby and I collected and some folks gave feedback on. I just tried to put it on the landing page without the general markup of the page exploding.

### CODE REVIEW

**Comment by dholbach** [comment]
Some observations and reactions to this:
- duplication between flux facts and gitops features section
- too many font-awesome icons on one page (I find too many of them start looking a bit bland)
- some of the flux facts feel a bit too wordy and like I'd just skim over them

**Comment by dholbach** [comment]
Screenshot from 2021-07-05 16-42-34

**Comment by dholbach** [comment]
A simple solution could be a table (I just applied some very quick styling - this could obviously be done a lot nicer):

Screenshot from 2021-07-05 16-47-42



What's good about this solution: it doesn't look as wordy any more and the colourful emoji look friendlier than the blue font-awesome icons.

**Comment by dholbach** [comment]
Feedback on both the text and the possible avenues for making this look great would be appreciated.

**Comment by scottrigby** [comment]
I like the table better! That was my initial thought too. Well, not a table but horizontal cards. But +💯  for the table!

**Comment by dholbach** [comment]
Screenshot from 2021-07-06 10-27-43

With a couple of CSS tweaks I got it to look like this ^, unfortunately this breaks mobile.

Maybe cards? https://www.docsy.dev/docs/adding-content/shortcodes/#card-panes

**Comment by dholbach** [comment]
Screenshot from 2021-07-06 11-42-26

Here's an attempt using `cardpane` and `card` - needs more styling to make it look nice though.

WDYT?

File: content/en/_index.html [hunk 1]
```diff
@@ -17,26 +17,49 @@
...
   The latest version of Flux brings many new features and made Flux more flexible and versatile.
   {{% /blocks/hero %}}
 
-
-  <div class="lead-gitops">
+  <div class="lead-flux">
   {{< blocks/lead >}}
 
-  <h2 id='gitops'>GitOps at your fingertips</h2>
+  <h2 id="flux-facts">Flux Facts!</h2>
```
Comment (hiddeco) [inline content/en/_index.html:26]:
```suggestion
  <h2 id="flux-facts">Flux in short</h2>
```

(and if not accepted: `Flux facts!`, as we do not use PascalCase title headings elsewhere on this page)

**Comment by dholbach** [comment]
Screenshot from 2021-07-07 14-53-55

Helpful feedback from @hiddeco ! @scottrigby seemed to learn towards the table design as well, so I'll go with that for now and try to make it look nicer.

**Review by scottrigby — APPROVED** [review]
Most recent change looks great on large and small screens, including mobile - nice work @dholbach! 👏 🙌 💪

**Comment by juozasg** [comment]
Looks good!

Agreed that some duplication with the Features section. Noticed the blue background for Resources section label is now the wrong color. I'll make a PR for that.
